src/diffusers/schedulers/scheduling_euler_ancestral_discrete.py

```diff
@@ -214,6 +247,8 @@ def step(
         elif self.config.prediction_type == "v_prediction":
             # * c_out + input * c_skip
             pred_original_sample = model_output * (-sigma / (sigma**2 + 1) ** 0.5) + (sample / (sigma**2 + 1))
+        elif self.config.prediction_type == "sample":
+            raise NotImplementedError("prediction_type not implemented yet: sample")
```

Comment:
On schedulers where I couldn't verify good outputs, I threw an explicit error that implies we should support prediction_type sample in the future. I'm not sure if this is correct or there's something inherent about the schedulers which makes them not support the sample prediction type.